frontend/.env.example

```
@@ -0,0 +1,7 @@
+# API base URL (set per environment)
+VITE_API_BASE_URL=http://localhost:8080
+
+# Auth settings
+VITE_AUTH_DOMAIN=https://auth.example.com
+VITE_AUTH_CLIENT_ID=your-client-id
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_AUTH_CLIENT_ID key should go before the VITE_AUTH_DOMAIN key

(UnorderedKey)